.github/workflows/ci.yaml

```diff
@@ -188,9 +188,9 @@ jobs:
 
       - name: Check auto generated files
         run: |
-          echo "Failing if any auto generated files are updated, checking 'git status'"
+          echo "Failing if any auto generated files are updated"
           git --no-pager diff
-          git status --porcelain 2>&1 | tee /dev/stderr | (! read)
+          git diff-index --quiet HEAD; echo $?
```

Comment:
The `; echo $?' is not needed - it is just for debugging locally to see that we get non zero exit code. But it is good that we have it since the code does not work yet:

```
Failing if any auto generated files are updated
0
```

The reason is that `make docker-build` does not generate the generated files. We need to move this job to the linters job.

The step should run:

```
echo "Checking that generated files were not modified"
make generate
git --no-pager diff
git diff-index --quiet HEAD
```